### ISSUE TITLE
ImageFetchFailure for pngs with a specific CDN? 

### ISSUE THREAD

**Issue by NoahFisher** [sdsykes/fastimage#56]
Hi, I am having issues gathering the sizes of png images from a specific cdn (other cdns work fine, so do other pngs, it's a strange combination of pngs coming from this cdn...). 

Sample url from not working cdn (note that the image is viewable if you copy and past into your browser): 

```
http://fbcdn-creative-a.akamaihd.net/hads-ak-xtf1/t45.1600-4/11410305_6023731601692_1599918971_n.png
```

Sample output from `FastImage.size` call: 

```
FastImage.size("http://fbcdn-creative-a.akamaihd.net/hads-ak-xtf1/t45.1600-4/11410305_6023731601692_1599918971_n.png", raise_on_failure:true, timeout:50)

FastImage::ImageFetchFailure: FastImage::ImageFetchFailure
```

Sample url from cdn that works properly: 
```
https://scontent.xx.fbcdn.net/hads-xfa1/t45.1600-4/10736922_6035637783473_440488633_n.png
```

Any ideas on where to start debugging?

**Comment by sdsykes** [comment]
I've seen images not be served because the server didn't like the ruby user agent.

In this case, I'm not sure what the problem is - it works here:

      FastImage.size "http://fbcdn-creative-a.akamaihd.net/hads-ak-xtf1/t45.1600-4/11410305_6023731601692_1599918971_n.png"
      => [1200, 627]

**Comment by sdsykes** [comment]
See here: https://github.com/sdsykes/fastimage/issues/54

**Comment by NoahFisher** [comment]
@sdsykes Thanks, that makes sense. 

Is there work being done to add the feature mentioned in #54 ? 
```Possibly I could add a feature so you could specify the user agent, I can consider it.```

**Comment by sdsykes** [comment]
It's fairly easy to do - I'd prefer to change the user agent to FastImage rather than add another configuration option.

Can you confirm that this is indeed the issue?

**Comment by sdsykes** [comment]
This issue was resolved by ad567ff0da748
